dashboard/static/src/js/components/Table.js

```diff
@@ -0,0 +1,126 @@
+import React from "react";
+import ReactTable from 'react-table-6';
+import {sendRequest, dateFormat} from './Utility';
+
+export default class Table extends React.Component {
+   constructor() {
+      super();
+      this.state = {
+        data: [],
+        loading: true,
+        pages: 0,
+        total: 0
+      };
+  }
+  componentDidMount() {
+    sendRequest("./Patient").then(response => {
+      let rawData = {};
+      if (response) {
+        rawData = JSON.parse(response);
+      }
+      let dataSet = [];
+      if (rawData["entry"]) {   
+        (rawData["entry"]).forEach(function(item) {
+            var nameNode = item.resource && item.resource.name ? item.resource.name[0] : null;
```

Comment:
Was just reading about "destructuring assignment" looks complicated, but maybe worth it for even more deeply nested FHIR?

https://blog.opendigerati.com/how-to-extract-nested-data-from-arrays-and-objects-with-one-line-of-code-using-destructuring-77f7fc25230c?gi=57355b814e64
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Destructuring_assignment

Reply:
that sounds great, ivan, will it still work for null object though, e.g. {a: name, b: phone} = obj where obj is null.  It seemed to break for me.

Reply:
Ah darn, I would've thought it could handle missing/null attributes